src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -11,16 +13,30 @@ schedules:
 trigger:
   batch: true
   branches:
-   include:
-   - release/*
-   - internal/release/*
+    include:
+    - main
+    - release/*
+    - internal/release/*
+
+pr:
+  branches:
+    include:
+    - main
+    - release/*
 
-pr: none
+variables:
+# enable source-only build for pipelines with the -source-build suffix
+- name: isSourceOnlyBuild
+  value: ${{ in(variables['Build.DefinitionName'], '-source-build') }}
 
 stages:
-- template: templates/stages/vmr-scan.yml
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'Schedule')) }}:

```

Comment:
I believe this was running all of the time prior to this refactoring.

Reply:
Yeah, this needs to be run for every branch in the full pipeline, not just main.